apps/src/code-studio/pd/components/scholarshipDropdown.jsx

```diff
@@ -4,7 +4,7 @@ import {FormGroup} from 'react-bootstrap';
 import Select from 'react-select';
 
 // update this to lock scholarships so that scholarship status can't be updated via the UI.
-const locked = true;
+const locked = false;
```

Comment:
This was part of shutting down last year's applications. I've gotten approval to unlock this.